app/controllers/users_controller.rb

```diff
@@ -77,11 +87,7 @@ def destroy
   end
 
   def show_schedule
-    if user_signed_in?
-      @sessions = current_user.tutoring_sessions
-    else
-      redirect_to new_user_session_path
-    end
+    @sessions = current_user.tutoring_sessions
```

Comment:
Pretty sure these deleted lines should be there

Reply:
Fixed.